### DISCUSSION TITLE
SEO img changing over builds.

### DISCUSSION THREAD

**Discussion by rroslaniec** [gatsbyjs/gatsby#34903] [Help]
Hey,

My problem is related to the fact that after each production build (e.g. after I add a new article to my site) I receive a totally new address (hash) to a cover image. It's very important because FB or LinkedIn do not refetch those images.

Folks, how do you deal with this problem? I thought that `original.src` will be the solution but not really :/

Maybe I missed some docs about generating a persist image address?

**Comment by roma-glushko** [comment]
I don't have the answer but it's concerning to me as well.
This is not only an issue from the social media perspective but also an SEO concern as people correctly was pointing out in:
- https://github.com/gatsbyjs/gatsby/issues/9676

@KyleAMathews how can we get stable image paths of generated images if the source images don't change?
Every time I deploy things my unchanged post hero image paths got changed. However, I have not changed the original files nor image generation configs at all. 

The deployment is as simple as 
- cleaning .cache/public directory 
- and the running `GATSBY_ENV=production gatsby build --prefix-paths`

I expect my generated image paths stay the same/stable if I have done nothing to them.

I'm using Gatsby 4.16.0 + gatsby-plugin-image 2.16.1
